js/Localization.js

```diff
@@ -0,0 +1,114 @@
+import i18n from 'i18n-js';
+import * as RNLocalize from 'react-native-localize';
+
+i18n.translations = {
```

Comment:
Could this be split up into multiple JSON files in order to allow an integration with Crowdin? https://github.com/zoontek/react-native-localize/blob/master/example/src/SyncExample.js makes me think that it is possible.